readme.md

```diff
@@ -1,24 +1 @@
-# ![bsp logo](resources/buildServerProtocol64.svg) Build Server Protocol
-
-[![Join the chat at https://gitter.im/scalacenter/bsp](https://badges.gitter.im/scalacenter/bsp.svg)](https://gitter.im/scalacenter/bsp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-This project is a work-in-progress effort to improve the integration
-between language server/editors and build tools. This effort is led by the
-Scala Center and Jetbrains, together with the help of other tool developers
-in the Scala ecosystem.
-
-Even though the protocol focuses on Scala developer tools, it's designed to
-be language-agnostic. We're looking for third parties in other programming
-languages to join this effort.
-
-Read the specification [here](docs/specification.md).
-
-## Maintainers
-
-The current maintainers (people who can merge pull requests) are:
-
-* Jorge Vicente Cantero - [@jvican](https://github.com/jvican)
-* Justin Kaeser - [@jastice](https://github.com/jastice)
-* Łukasz Wawrzyk - [@lukaszwawrzyk](https://github.com/lukaszwawrzyk)
-* Ólafur Páll Geirsson - [@olafurpg](https://github.com/olafurpg)
-* Tomasz Pasternak - [@tpasternak](https://github.com/tpasternak)
+### ![bsp logo](resources/buildServerProtocol64.svg) Build Server Protocol
```

Comment:
I don't think this is intentional?

Reply:
Ahh yeah my mistake, I was playing around after making the pull request and didn't think that would be included. Kind of a noob with this type of stuff still. How would you like me to resolve this? I can change the readme back or close this out and open a new one, I dunno is there's a fancier git way to do this.

Reply:
You can just change it and push, and I'll squash on merge. Or you can reset the branch and force-push to get rid of the bad commits.

Reply:
Okay, it should be good to go now. Thanks for the help and good luck with this super cool project!